### PR TITLE
Fix build command for Docker images.

### PR DESCRIPTION
**Pull request by spikecurtis**
Doc update for how to build Docker images.

### CODE REVIEW

**Comment by spikecurtis** [comment]
I signed it.

File: doc/testing.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ Istio Pilot runs end-to-end tests as part of the presubmit check. The test drive
 
 ## Docker images
 
-The following Bazel command generates Docker images for the proxy agent and proxy container:
+The following Bazel command generates Docker images for the proxy agent and proxy container, then pushes them to a repository:
 
-    bazel run //docker:runtime
+    ./bin/push-docker --hub=gcr.io/your-testing-hub
```
Comment (ldemailly) [inline doc/testing.md]: here:
```
./bin/push-docker --hub=your-testing-hub
```

**Comment by kyessenov** [comment]
force merged since this is just a doc update.

**Pull request merged**